src/libcore/num/mod.rs

```diff
@@ -2328,7 +2328,7 @@ macro_rules! uint_impl {
         }
 
         /// Returns the smallest power of two greater than or equal to `self`.
-        /// Unspecified behavior on overflow.
+        /// More details about overflow behavior can be found in [RFC 560].
```

Comment:
Hm, this doesn't match what was discussed in #18604, specifically https://github.com/rust-lang/rust/issues/18604#issuecomment-225058456 indicates it should panic in debug mode.